src/components/Modals/UploadDocumentModal.jsx

```diff
@@ -133,6 +139,7 @@ const UploadDocumentModal = ({ showModal, setShowModal }) => {
         onConfirm={confirmationModalType === 'add' ? handleDocAdd : handleDocReplace}
         uploadType={confirmationModalType}
       />
+      km
```

Comment:
There's an additional `km` string in the modal (see upper left)

![Screenshot 2024-04-29 at 11 30 08 AM](https://github.com/codeforpdx/PASS/assets/14917816/70b19777-868b-43aa-8645-dcc93b281fcf)